src/app/form-builder/form-builder.component.scss

```
@@ -0,0 +1,123 @@
+h2,
+h3 {
+   text-align: center;
+   margin-top: 10px;
+}
+
+.control-form {
+   display: flex;
+   justify-content: flex-start;
+   button {
+      margin: 0px 10px;
+   }
+}
+
+form {
+   border: solid 1px #ccc;
```

create _variables.scss and use color or font variables from there
```suggestion
   border: solid 1px $grey;
```